tests/first.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import first from '../src/first';
+
+describe( 'utils', () => {
+	describe( 'first', () => {
+		it( 'should return first item', () => {
+			const iterator = [ 11, 22 ][ Symbol.iterator ]();
```

Comment:
Good point. It took me a couple of secs to understand what the `[][ Symbol.iterator()` does :P.